bin/run_e2e_tests

```diff
@@ -17,17 +17,33 @@ print_error () {
 }
 
 setup () {
+    ${dockerComposeCmd} up -d db # start database first - it takes about 10 sec untils db startsup, and otherwise db migration fails
+    if [ ! $? -eq 0 ]; then
+        print_error "Db startup failed"
+        cleanup
+        exit 1
+    fi
+
     ${dockerComposeCmd} build
```

Comment:
What if images are already built, won't this finish instantly while database is not set-up yet?